newsModel.js

```diff
@@ -0,0 +1,51 @@
+class NewsModel {
+  constructor() {
+    this.headlines = [];
```

Comment:
I'd suggest maybe having a single array, e.g `this.news` in the model, which would be an array of JS objects (either plain key-value objects, or instances of a model class) having each the properties `headline`, `link` and `image`, to make the data more cohesive (and avoiding inconsistent data, for example having the headlines in an order mismatched with the order of the links array, etc)